config/i18n-tasks.yml

```diff
@@ -132,6 +132,7 @@ ignore_unused:
   - 'reports.new.categories.*' # double interpolation in reports_helper
   - 'shared.pagination.*'
   - 'auth.providers.*'
+  - 'changesets.paging_nav.*' # used by string interpolation in elements pagination
```

Comment:
Won't be required if #4872 is merged, although it uses the same effect as in https://github.com/openstreetmap/openstreetmap-website/pull/5223/files#r1766713701 to some extent.